Templates/Tests/Expected/AutoMockable.expected

```diff
@@ -125,11 +145,12 @@ class ReservedWordsProtocolMock: ReservedWordsProtocol {
     }
     var continueWithReceivedMessage: String?
     var continueWithReturnValue: String!
+	var continueWithClosure: ((String) -> String)?
 
     func `continue`(with message: String) -> String {
         continueWithCallsCount += 1
         continueWithReceivedMessage = message
-        return continueWithReturnValue
+        return continueWithClosure != nil ? continueWithClosure!(message) : continueWithReturnValue
```

Comment:
how about adjusting the template so that you don't need to force unwrap it ?
```swift
continueWithClosure.map({ continueWithClosure(message) }) ?? continueWithReturnValue
```